tests/p2p_sync_throttle_test_shape.json

```diff
@@ -0,0 +1,142 @@
+{
```

Comment:
Why is this shape file needed? Can you not configure what you need without it?

Reply:
The line topology has nodes connecting in ascending order and launcher only supports the highest numbered nodes being unstarted.  Since throttling is only applicable to listen ports, I had to use a custom topology so the highest numbered node is connecting into the second-highest node where the throttle is applied.